modules/gcp/storage-classes/stage1/main.tf

```diff
@@ -1,3 +1,18 @@
+resource "kubernetes_storage_class" "fast" {
+  metadata {
+    name = "fast"
+  }
+
+  allow_volume_expansion = true
+  storage_provisioner    = "pd.csi.storage.gke.io"
+  reclaim_policy         = "Delete"
+  volume_binding_mode    = "Immediate"
+
+  parameters = {
+    "type" = "pd-balanced"
```

Comment:
These are not very fast. I am going to guess they should be `pd-ssd` or if these should be `pd-balanced` then the `gp` class below should be `pd-standard`